.travis.yml

```diff
@@ -22,12 +22,15 @@ install:
   - if [ $RESULT -eq 0 ]; then git checkout -f HEAD ; fi
   - rm `git status | grep "\.lean" | sed "s/\.lean/.olean/"` ||  true
   - rm `git status | grep "\.lean"` || true
+  - ./purge_olean.sh
   - rm mathlib.txt || true
 
 jobs:
   include:
     - stage: Pre-build-1
       script:
+        - find . -name "*.olean" -delete
+        - rm -rf src/category_theory/examples/
```

Comment:
is this still necessary? Are there more general ways, like removing all empty directories?

Reply:
I'm still not sure what the issue is. The directory has been deleted but it seems like it linger in Travis' cache. I'm still trying to figure out this issue. The build fails but it works on my computer.

Reply:
I guess I found it: your change contains a dangling file: `src/category_theory/limits/limits/over.lean` I guess if you remove it the test should work